metricbeat/metricbeat.full.yml

```diff
@@ -90,6 +90,38 @@ metricbeat.modules:
   # Password of hosts. Empty by default
   #password: test123
 
+#-------------------------------- ceph Module --------------------------------
+- module: ceph
+  metricsets: ["perf"]
+  enabled: true
+  period: 1s
+
+ceph:  
+  # location of ceph binary
+  binary_path = "/usr/bin/ceph"
+  #
+  #####
+  ### MetricSet perf gather statistics via the admin socket, requires socket_dir, mon_prefix, osd_prefix and socket_suffix
+  #####
+  # directory in which to look for socket files
+  socket_dir = "/var/run/ceph"
+  # prefix of MON and OSD socket files, used to determine socket type
+  mon_prefix = "ceph-mon"
+  osd_prefix = "ceph-osd"
+  # suffix used to identify socket files
+  socket_suffix = "asok"
+  #
+  #####
+  ### MetricSets status,df and poolio gather statistics via ceph commands, requires ceph_user and ceph_config
+  #####
+  # Ceph user to authenticate as, ceph will search for the corresponding keyring
+  # e.g. client.admin.keyring in /etc/ceph
+  # Consult the ceph documentation for more detail on keyring generation.
+  user = "client.admin"
+  # Ceph configuration to use to locate the cluster
+  config_path = "/etc/ceph/ceph.conf"
```

Comment:
"...means only one of the instances connecting needs to collect them?" Yes.
"If there are multiple MON, will they provide different data?" No, at least it's not expected this behavior.